test/manager.js

```diff
@@ -1117,30 +1117,6 @@ describe('Manager', () => {
             expect(rendered).to.contain('<h1>default message</h1>');
         });
 
-        it('renders with a global context function that returns a promise (no request)', async () => {
```

Comment:
Turns out this func wasn't required for 100% code coverage. The fact that we're using async/await is Node dependent